packages/api-reference/src/components/api-client-bus.ts

```diff
@@ -5,3 +5,4 @@ const apiClientBusKey: EventBusKey<OpenClientPayload> = Symbol()
 
 /** Event bus to open the API Client */
 export const apiClientBus = useEventBus(apiClientBusKey)
+export const modalStateBus = useEventBus('modalState')
```

Comment:
lets call this searchModalState

Reply:
being the api client modal state, should we still use search here?

Reply:
ah lets just keep it nvm : )